language-plutus-core/src/Language/PlutusCore/Core/Type.hs

```diff
@@ -159,9 +160,10 @@ deriving newtype instance PrettyBy config a => PrettyBy config (Normalized a)
 type family HasUniques a :: Constraint
 type instance HasUniques (Kind ann) = ()
 type instance HasUniques (Type tyname ann) = HasUnique (tyname ann) TypeUnique
-type instance HasUniques (Term tyname name ann) =
-    (HasUnique (tyname ann) TypeUnique, HasUnique (name ann) TermUnique)
-type instance HasUniques (Program tyname name ann) = HasUniques (Term tyname name ann)
+type instance HasUniques (Term tyname name ann)
+    = (HasUnique (tyname ann) TypeUnique, HasUnique (name ann) TermUnique)
+type instance HasUniques (Program tyname name ann) = HasUniques
+    (Term tyname name ann)
```

Comment:
Kind of weird formatting. Are you using some tool?

Reply:
I'm sometimes running stylish haskell over it.